### ISSUE TITLE
Nested SCSS selectors starting with a combinator are not recognised

### ISSUE THREAD

**Issue by mattrussell-sonocent** [malinindev/check-unused-css#86]
A class defined in a nested selector that starts with `>`, `+` or `~` is reported as non-existent.

### Example

**`src/Foo.tsx`**
```tsx
import styles from './Foo.module.scss'

export const Foo = () => <div className={styles.wrapper}><span className={styles.item} /></div>
```

**`src/Foo.module.scss`**
```scss
.wrapper {
  > .item {
    color: red;
  }
}
```

```
$ check-unused-css src

Found 1 classes used in source files but non-existent in CSS:
  Foo.tsx:3:82 - .item
```

Tested on version 0.5.1.

**Comment by malinindev** [comment]
@mattrussell-sonocent, thank you for the report, fixed in v5.2

**Comment by mattrussell-sonocent** [comment]
> [@mattrussell-sonocent](https://github.com/mattrussell-sonocent), thank you for the report, fixed in v5.2

Thanks!
